src/platform/packages/private/kbn-grid-layout/grid/grid_row/grid_row.tsx

```diff
@@ -63,11 +61,11 @@ export const GridRow = forwardRef<HTMLDivElement, GridRowProps>(
         /** Update the styles of the grid row via a subscription to prevent re-renders */
         const interactionStyleSubscription = combineLatest([
           gridLayoutStateManager.interactionEvent$,
-          gridLayoutStateManager.gridLayout$,
           gridLayoutStateManager.runtimeSettings$,
+          gridLayoutStateManager.gridLayout$,
```

Comment:
I am wondering if we can remove this since it's not being used. So far, I've just placed it at the end of the subscriptions array. I think it's safe to remove, but I’d appreciate it if you could double-check @Heenawter 🙏🏼